### DISCUSSION TITLE
tv.vim - a vim plugin for television

### DISCUSSION THREAD

**Discussion by prabirshrestha** [alexpasmantier/television#113] [Show and tell]
If folks are interested in using television inside vim/neovim feel free to give [tv.vim](https://github.com/prabirshrestha/tv.vim) a try. It is currently at a very experimental phase. There are some bugs that I will iron out when I get time in the upcoming days.

![tv.vim](https://github.com/user-attachments/assets/c2a1dc2a-55d0-4b7b-9ac9-f04a5ac0cb33)

**Comment by martinengelke** [comment]
Could you try to implement a pipe for searching text in the e.g. CWD, so that we can make use of that: 

https://github.com/alexpasmantier/television/discussions/326#discussioncomment-12289574

**tv** then will just **jump** directly to the occurrence of your search phrase!
